plugin/acl/acl.go

```diff
@@ -96,7 +99,19 @@ RulesCheckLoop:
 func matchWithPolicies(policies []policy, w dns.ResponseWriter, r *dns.Msg) action {
 	state := request.Request{W: w, Req: r}
 
-	ip := net.ParseIP(state.IP())
+	ip := []byte(nil)
```

Comment:
```suggestion
	var ip net.IP
```

Reply:
Just for my knowledge, is there an actual difference between using the var keyword and the short declaration operator in this particular case ?

Changing []byte to net.IP, I understand is safer.